docs/developer-center/new-guides/quickstart.md

```diff
@@ -1,185 +1,966 @@
 
-![Bikeshare data](../img/new-guides/quickstart/explore_layer.png)
+<div class="example-map">
+    <iframe
+        id="quickstart_guide_starbucks_brooklyn"
+        src="https://cartoframes.carto.com/kuviz/f442503a-f612-403d-b171-95e9138d0188"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+</div>
+
 
 Great! We have a map!
 
-Now, that you have a better sense of about where the stations are located, it's time to continue with the exploration. The next question to answer is which stations around the city are most active. To visualize this on the map, you can use a CARTOframes layer helper. Using the column `total_events`, which is the number of dropoffs and pickups at each station for the past month, let’s try a visualization helper called `size_continuous_layer` to size each station by that value:
+Now, you have a better sense about where the stores are. To continue with your exploration, you want to know where are the stores with more revenue. To do so, you can use the `size_continuous_layer` visualization layer:
+
 
-```py
+```python
 from cartoframes.viz.helpers import size_continuous_layer
 
-Map(size_continuous_layer(bikeshare_df, 'total_events'))
+Map(size_continuous_layer(starbucks_df, 'revenue', 'Revenue in $'))
 ```
-![Bikeshare data](../img/new-guides/quickstart/explore_helper.png)
 
-Good job! Now, just taking a look, you can see where are the stations with more activity. Also, thanks to be using a helper, we get a legend out of it.
+<div class="example-map">
+    <iframe
+        id="quickstart_guide_starbucks_brooklyn_by_revenue"
+        src="https://cartoframes.carto.com/kuviz/2e27384e-0cb4-4c02-809f-efa618ee5a6c"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+</div>
+
+Good job! Now, just taking a look, you can see where are the stores with more revenue. Also, thanks to be using a helper, we get a legend out of it.
 
 To learn more about visualizating your data, about how to add legends, pop-ups, widgets and how to do it faster thanks to helpers, check the [visualization guide]().
 
+## Create your areas of influence
 
-### Discover and enrich data thanks to the CARTO catalog
+In the same way as with the geocoding, you can use a straightforward method for creating your areas of influence using isochrones. Isochrones are lines drawn on a map connecting points at which something occurs or arrives at the same time.
 
-You already know where your company stations are and their activity, now you want to know if they are in optimal locations. You start thinking about which data could be valuable to validate that and decide to check if there can be any correlation with households with no car data. Let's see how CARTOframes can help you finding that data and enrich yours.
+For our analysis, let's create and plot isochrones of 15 minutes by walk. Let's see how to do it:
```

Comment:
"For our analysis, let's create isochrones for each store that cover the area within a 15 minute walk. 

To do this we will use the Isolines data service: